shap.Rmd

```diff
@@ -27,3 +27,5 @@ ex.
 
 ### Examples
 
+
+**NOTE: DO NOT USE THE `SHAPPER` PACKAGE. IT IS AWFUL**
```

Comment:
Can you be more specific about what the issue is?

Reply:
tl;dr About a 2.36/10 (and that’s being generous). The dependencies are no longer available on CRAN and the example code provided does not work (see [here](https://github.com/ModelOriented/shapper))

More specifically, the fact that this package requires you to install Python dependencies to then wrap R around those Python functions seems rather silly. The only possible use I could see for this is if you are a Python developer who refuses to learn R and needs to use shapper. But at that point, why not do the project in Python itself? If you have to migrate to R, there are a lot of other SHAP packages available that work great (see shapr for example).

If you insist on using this package, I am sorry for your loss and hope you don’t have too many hours of unpaid overtime left. Godspeed ahead.